protocol/cpp/include/solarxr_protocol/generated/all_generated.h

```diff
@@ -6,6 +6,13 @@
 
 #include "flatbuffers/flatbuffers.h"
 
+// Ensure the included flatbuffers.h is the same version as when this file was
+// generated, otherwise it may not be compatible.
+static_assert(FLATBUFFERS_VERSION_MAJOR == 2 &&
+              FLATBUFFERS_VERSION_MINOR == 0 &&
+              FLATBUFFERS_VERSION_REVISION == 6,
+             "Non-compatible flatbuffers version included");
```

Comment:
why does this keep getting added and removed? could we all please use the same damn version of flatc?

Reply:
i use the one in the repo .... that one should be the default

Reply:
stop resolving my discussions.